packages/gatsby/src/utils/flags.ts

```diff
@@ -172,16 +172,6 @@ const activeFlags: Array<IFlag> = [
     umbrellaIssue: `https://gatsby.dev/parallel-sourcing-feedback`,
     testFitness: (): fitnessEnum => true,
   },
-  {
-    name: `FUNCTIONS`,
-    env: `GATSBY_EXPERIMENTAL_FUNCTIONS`,
-    command: `all`,
-    telemetryId: `Functions`,
-    experimental: false,
-    description: `Compile Serverless functions in your Gatsby project and write them to disk, ready to deploy to Gatsby Cloud`,
-    umbrellaIssue: `https://gatsby.dev/functions-feedback`,
-    testFitness: (): fitnessEnum => true,
-  },
```

Comment:
Removing it will cause failures for folks that have flag in config (unknown flag message) - should we keep this flag (similar to QUERY_ON_DEMAND) to avoid causing problems like that?

Reply:
Oh! I didn't know that that was the case. Makes sense. I'll add it back! Do we print a message for QUERY_ON_DEMAND to say that it is now enabled by default?

Reply:
Okay, just found this 

> * LOCKED_IN means that feature is enabled always (unless `noCI` condition is met).
> * This is mostly to provide more meaningful terminal messages instead of removing
> * flag from the flag list when users has the flag set in configuration
> * (avoids showing unknown flag message and shows "no longer needed" message).

Excellent, I'll add it back and set to `LOCKED_IN`

Reply:
Done!